### ISSUE TITLE
Nebula Logger: Illegal assignment from List<Schema.PicklistEntry> to List<PicklistEntry>

### ISSUE THREAD

**Issue by aranwe** [jongpie/NebulaLogger#272]
Hi,

when trying to install Nebula Logger I get the following error:
```
1.  Illegal assignment from List<Schema.PicklistEntry> to List<PicklistEntry>
LogHandler: Illegal assignment from List<Schema.PicklistEntry> to List<PicklistEntry>

2.  Method does not exist or incorrect signature: void getValue() from the type PicklistEntry
LogHandler: Method does not exist or incorrect signature: void getValue() from the type PicklistEntry

3.  Method does not exist or incorrect signature: void getValue() from the type PicklistEntry
LogHandler: Method does not exist or incorrect signature: void getValue() from the type PicklistEntry

4.  Dependent class is invalid and needs recompilation:
 Class LogHandler : Illegal assignment from List<Schema.PicklistEntry> to List<PicklistEntry>
LogHandler_Tests: Dependent class is invalid and needs recompilation:
 Class LogHandler : Illegal assignment from List<Schema.PicklistEntry> to List<PicklistEntry>

5.  Invalid type: LogHandler
Log: Invalid type: LogHandler

6. (LoggerLogManagement) No classes found for LogHandler_Tests.
LoggerLogManagement: No classes found for LogHandler_Tests.
```

I have actually noticed that Salesforce has recently changed the return type of getPicklistValues() and started to throw type mismatch on Schema vs global PicklistEntry.

I can make PR if needed :) 

Thank you!

**Comment by jongpie** [comment]
Hey @aranwe, thanks for reporting this! Do you know which release your org is currently using? I'm not able to reproduce this error, so I'm wondering if it's possibly related to the new Spring '22 release (assuming that's the version your org is using).

**Comment by aranwe** [comment]
Yup, it's a sandbox with Spring '22 -> it's possible that it's connected, I
was just going through the Release notes and couldn't find any mention of
either PicklistEntry class or anything that seemed related to this in the
Schema namespace.

**Comment by jongpie** [comment]
Sounds good, thanks for confirming! I haven't done much testing in a Spring '22 org yet, but I'll see if I can create one & test this issue. It should be a fairly small code change, but I'd like to be able to reproduce the issue before making any code changes. I'll keep you updated on the status, but I have several other bugfixes in-progress, so I'll probably put out another release in a few weeks that contains several bugfixes (including for this issue).

**Comment by jongpie** [comment]
@aranwe I've fixed this bug in the new release, [v4.7.0](https://github.com/jongpie/NebulaLogger/releases/tag/v4.7.0).
